
<html class="no-js" lang="en">


<head>

    <!-- mobile specific metas
   ================================================== -->
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <!-- CSS
   ================================================== -->
    <link rel="stylesheet" href="https://2021.igem.org/wiki/index.php?title=Template:Phystech_Moscow/base.css&action=raw&ctype=text/css">
    <link rel="stylesheet" href="https://2021.igem.org/wiki/index.php?title=Template:Phystech_Moscow/vendor.css&action=raw&ctype=text/css">
    <link rel="stylesheet" href="https://2021.igem.org/wiki/index.php?title=Template:Phystech_Moscow/main.css&action=raw&ctype=text/css">
    <link rel="stylesheet" href="https://2021.igem.org/wiki/index.php?title=Template:Phystech_Moscow/mine.css&action=raw&ctype=text/css">


    <!-- script
   ================================================== -->

    <script type="text/javascript" src="https://2021.igem.org/wiki/index.php?title=Template:Phystech_Moscow/modernizr.js&action=raw&ctype=text/javascript"></script>
    <script type="text/javascript" src="https://2021.igem.org/wiki/index.php?title=Template:Phystech_Moscow/pace.min.js&action=raw&ctype=text/javascript"></script>


    <!-- favicons
	================================================== -->
    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
    <link rel="icon" href="favicon.ico" type="image/x-icon">

</head>

<body id="top">

    <header id="header" class="row">   

		<div class="header-logo">
		<a href="https://2021.igem.org/Team:Phystech_Moscow">Team</a>
		</div>

		<nav id="header-nav-wrap">
			<ul class="header-main-nav">
				<li class="current"><a class="smoothscroll"  href="#home" title="home">Home</a></li>
				<li><a class=""  href="#" title="team">Team</a></li>
				<ul>
					<li><a href="https://2021.igem.org/Team:Phystech_Moscow/Team">About team</a></li>
					<li><a class="imp1" href="https://2021.igem.org/Team:Phystech_Moscow/Attributions">Attributions</a></li>
					<li><a class="imp2" href="https://2021.igem.org/Team:Phystech_Moscow/Collaborations">Collaborations</a></li>
				</ul>
				<li><a class=""  href="#" title="project">Project</a></li>
				<ul>
					<li><a class="imp3" href="https://2021.igem.org/Team:Phystech_Moscow/Communication">Communication</a></li>
					<li><a class="imp1" href="https://2021.igem.org/Team:Phystech_Moscow/Contribution">Contribution</a></li>
					<li><a class="imp1" href="https://2021.igem.org/Team:Phystech_Moscow/Description">Description</a></li>
					<li><a class="imp2" href="https://2021.igem.org/Team:Phystech_Moscow/Engineering">Engineering</a></li>
					<li><a class="imp3" href="https://2021.igem.org/Team:Phystech_Moscow/Model">Model</a></li>                   
					<li><a href="https://2021.igem.org/Team:Phystech_Moscow/Experiments">Experiments</a></li>
					<li><a class="imp2" href="https://2021.igem.org/Team:Phystech_Moscow/Implementation">Implementation</a></li>
					<li><a class="imp3" href="https://2021.igem.org/Team:Phystech_Moscow/Proof_Of_Concept">Proof of сoncept</a></li>
					<li><a href="https://2021.igem.org/Team:Phystech_Moscow/Results">Results</a></li>
					<li><a class="imp3" href="https://2021.igem.org/Team:Phystech_Moscow/Partnership">Partnership</a></li>
				</ul>
				<li><a class=""  href="https://2021.igem.org/Team:Phystech_Moscow/Safety" title="safety">Safety</a></li>
				<li><a class="imp23"  href="https://2021.igem.org/Team:Phystech_Moscow/Human_Practices" title="human_practices">Human practices</a></li>	
			</ul>
			<a href="https://2021.igem.org/Team:Phystech_Moscow/Results" class="button button-primary cta">
				Try it!
			</a>
		</nav>

 		<a class="header-menu-toggle" href="#"><span>Menu</span></a>    	

	</header> <!-- /header -->


   <!-- home
   ================================================== -->
   <section id="home" data-parallax="scroll" data-image-src="https://2021.igem.org/wiki/images/5/5b/T--Phystech_Moscow--Moscow.jpg" data-natural-width=3000 data-natural-height=2000>

        <div class="overlay"></div>
        <div class="home-content">        

            <div class="row contents">                     
                <div class="home-content-left">

                    <h3 data-aos="fade-up">Phystech_Moscow</h3>

                    <h1 data-aos="fade-up">
                        Engineering
                    </h1>
                </div>
            </div>
            <ul class="home-social-list">
				<li>
					<a href="https://www.instagram.com/igem_phystech/" target="_blank"><i class="fa fa-instagram"></i></a>
				</li>
				<li>
					<a href="https://web.facebook.com/igemphystech" target="_blank"><i class="fa fa-facebook-square"></i></a>
				</li>
			</ul>

            <div class="home-scrolldown">
				<a href="#styles" class="scroll-icon smoothscroll">
					<span>Down</span>
					<i class="icon-arrow-right" aria-hidden="true"></i>
				</a>
			</div>

        </div> <!-- end home-content -->
    </section>

    <div>
        <p id="img_license" style="padding-left:5% ;font-size: 0.9rem;font-style: italic;"><a href="https://www.flickr.com/photos/95572727@N00/48475349166">"Downtown Moscow"</a><span> by <a href="https://www.flickr.com/photos/95572727@N00">Trey Ratcliff</a></span> is licensed under <a href="https://creativecommons.org/licenses/by-nc-sa/2.0/?ref=ccsearch&atype=html" style="margin-right: 5px;">CC BY-NC-SA 2.0</a></p>
    </div>

    
   <!-- styles
   ================================================== -->
    <section id="styles">
		
        <div class="row narrow add-bottom text-center">

            <div class="col-twelve tab-full">

                <!-- <h1 data-aos="fade-up">This is the style page.</h1> -->

                <p data-aos="fade-up" class="lead">Our goal was to create a model predicting interaction between miRNA and lncRNA. This task is not new for scientists, but still there is not any model that can predict interactions well. So we decided to think about why it can be that Olga says there is not any good model. Maybe the existing models are just not available for Olga’s usage? Or maybe the models learnt are overfitted to the datasets the scientists used for model creation?

                </p>   			

            </div>

        </div>

        <div class="row" data-aos="fade-up">

            <div class="col-six tab-full">

                <h3 style="color:white">Machine learning approach overview</h3>
                <p>
                    Machine learning is a set of methods based on finding statistical relationships between a target variable and features. These relationships are sought in the form of some distribution function that depends on parameters. Parameters are initialized with some random values. The learning process is the search for such parameter values ​​that the function most accurately repeats the type of dependence of the target variable on features. For this search, some kind of metric of correspondence between the true function and the one that we have at the moment is specified, then the search for the optimal values ​​of the parameters is carried out by the method of gradient descent in the parameter space.
                    The prediction task in machine learning implies the following standard pipeline:
                    <br/>1) investigation of the problem, scientific area, recent researches and solutions of this problem
                    <br/>2) finding data sources for a task, collecting data and forming a dataset of positive and negative examples
                    <br/>3) exploratory data analysis -- close study of collected data: drawing distribution of feature values, searching for some simple correlations, filling of empty values, filtering out outliers
                    <br/>4) sequential examining of different model architectures (the kinds of function that depends on parameters) in order to estimate which model will perform best for this particular task, then training the model over a sufficient number of epochs (so that the estimated accuracy reaches a plateau)
                    <br/>5) deployment of the model so that it can be used without going into implementation details -- this point is often ignored by bioinformaticians, so their models cannot be used by biologists working in wet labs
                    
                </p>

                
                <h3 style="color:white">In relation to our task</h3>
                <p>
                    1) Problem to solve: lncRNA and miRNA are molecules that are situated in the cytoplasm during interaction. The interaction is carried out through the Ago protein. In addition to their paired interaction, lncRNA and miRNA also interact with  proteins and mRNA. The effectiveness of the interaction depends on the local concentrations in a given cell compartment, as well as on the affinity of the interaction, which depends on sequence complementarity, 2D and 3D structure.
                    <br/>2) Which sources of data are available: to find out and validate the interaction qPCR methods and various types of sequencing after enrichment for interacting regions ( HITS-CLIP, PAR-CLIP) are used. Information about such experiments is published in scientific articles, then collected in various databases that are used by scientists from all over the world to check the RNA of interest. Also, various indirect data can be added to the model as features, which is desirable to be easy to obtain for any pair of miRNA-lncRNA, so that the model can be used to predict the interaction for as many lncRNA and miRNA as possible. Such data can be levels of expression, secondary structure, information on interactions with various proteins. Expression data can reveal interactions if some correlations in the expression of interacting RNAs can be seen in the data. Secondary structure and interaction with proteins can influence the affinity of the interaction.
                </p>


            </div>

            <div class="col-six tab-full">


                <p> 3) Data usage in our research: We chose the lncRNASNP2 and DIANAv3 databases as the data sources for the target variable of our model (that is, the interaction facts). The lncRNA sequences taken from the ensembl database, miRNA sequences from the mirBase database were used as input data, and the secondary structure was calculated using the RNAfold program. Since the databases mainly contain information about positive examples of interaction, the negative examples needed to train the model had to be constructed independently. We did it in the following way: to be sure that sampled pair of miRNA and lncRNA does not interact, we not only required that this pair is not presented in any database of positives,  . It should be specially noted here that a good data sample is the cornerstone of building the ML model, it is said: garbage in - garbage out. In order to correctly form a sample for our problem, it is necessary to have competence both in the field of molecular biology and in the field of applied mathematics. Specifically, for this task, it is necessary that lncRNA and miRNA in positive and negative pairs do not differ significantly, since otherwise the model will tend to overfit (just memorize the answer for a pair according to which lncRNA and miRNA are involved in it, and not look for patterns that facilitate interaction)
                    <br/>4) In order to try to understand what actually determines the interaction of lncRNA and miRNA, we evaluated the rates of alignment of miRNA and lncRNA by groups of positive and negative examples, as well as the correlation coefficients of expression levels. To our surprise, no significant difference was found (PICTURES_HERE), from which we concluded ... retraining. Expression data was only used to filter negative examples as described above.
                    <br/>5) We have chosen a combination of CNN and RNN as the architecture of the model, as they are well suited for this task. CNN can capture local patterns of interaction, and RNN helps to detect colocalization of these patterns and their mutual reinforcement. The architecture of our model is clearly shown in the figure PICTURE_HERE
                    <br/>6) Made the model predictions available from the wiki
                    <br/>7) Target location prediction:
                    <br/>- Unet segmentation: Based on the clip seq analysis data, the interaction coordinates relative to the lncRNA transcript were found and sequences of these interaction sites were obtained. The lncRNA sequence was encoded (one hot encoder) and cut into fragments, a microRNA sequence was added to each fragment that interacts with this lncRNA. The architecture consisted of 2 encoder layers (3 convolutional layers and max pooling). During the training process, the model tended to predict only class 0, the redistribution of weights for classes when calculating the error and 1000 training epochs did not change the result
                    <br/>- DNAbert: an NLP model from the bert family (fully connected models with 
                    </p>                    
            </div>	         

        </div> <!-- end row -->
 
        <div class="row" data-aos="fade-up" style="align-content: right; padding-right: 5%;">
            <a style="margin-left: auto; margin-right: 0;" class="button button-primary" href="#">Next</a>
        </div>
    </section>


    <!-- footer
    ================================================== -->
    <footer>

        <div class="footer-bottom">
            <div class="row">
                <ul class="footer-social-list">
                    <li>
                        <p>Phystech_Moscow</p>
                    </li>
                    <li>
                        <a href="https://www.instagram.com/igem_phystech/" target="_blank"><i class="fa fa-instagram"></i></a>
                    </li>
                    <li>
                        <a href="https://web.facebook.com/igemphystech" target="_blank"><i class="fa fa-facebook-square"></i></a>
                    </li>
                </ul>
            </div>

            <div class="row">

                <div class="col-twelve">
                    <div class="copyright">
                        <span>© Copyright Phystech_Moscow 2021</span> 
                        <span>© Copyright Dazzle 2018</span> 
                        <span>Design by <a href="http://www.styleshout.com/">styleshout</a></span>		         	
                    </div>

                    <div id="go-top">
                        <a class="smoothscroll" title="Back to Top" href="#top"><i class="icon-arrow-up"></i></a>
                    </div>         
                </div>

            </div> <!-- end footer-bottom -->     	

        </div>

    </footer>

    <script type="text/javascript" src="https://2021.igem.org/wiki/index.php?title=Template:Phystech_Moscow/jquery-2.1.3.min.js&action=raw&ctype=text/javascript"></script>
    <script type="text/javascript" src="https://2021.igem.org/wiki/index.php?title=Template:Phystech_Moscow/plugins.js&action=raw&ctype=text/javascript"></script>
    <script type="text/javascript" src="https://2021.igem.org/wiki/index.php?title=Template:Phystech_Moscow/main.js&action=raw&ctype=text/javascript"></script>
    <script type="text/javascript" src="https://2021.igem.org/wiki/index.php?title=Template:Phystech_Moscow/mine.js&action=raw&ctype=text/javascript"></script>
    <script type="text/javascript">
        AOS.init();
    </script>
    <script type="text/javascript">
        $('#top_title').remove();
        $('#HQ_page').removeAttr('id');
        $('#content').removeAttr('id');
        $('#globalWrapper').removeAttr('id');
        $('#bodyContent').removeAttr('id');
        $('#mw-content-text p').css('margin-bottom', '0px');
    </script>
</body>

</html>