
<html class="no-js" lang="en">


<head>

    <!-- mobile specific metas
   ================================================== -->
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <!-- CSS
   ================================================== -->
    <link rel="stylesheet" href="https://2021.igem.org/wiki/index.php?title=Template:Phystech_Moscow/base.css&action=raw&ctype=text/css">
    <link rel="stylesheet" href="https://2021.igem.org/wiki/index.php?title=Template:Phystech_Moscow/vendor.css&action=raw&ctype=text/css">
    <link rel="stylesheet" href="https://2021.igem.org/wiki/index.php?title=Template:Phystech_Moscow/main.css&action=raw&ctype=text/css">
    <link rel="stylesheet" href="https://2021.igem.org/wiki/index.php?title=Template:Phystech_Moscow/mine.css&action=raw&ctype=text/css">


    <!-- script
   ================================================== -->

    <script type="text/javascript" src="https://2021.igem.org/wiki/index.php?title=Template:Phystech_Moscow/modernizr.js&action=raw&ctype=text/javascript"></script>
    <script type="text/javascript" src="https://2021.igem.org/wiki/index.php?title=Template:Phystech_Moscow/pace.min.js&action=raw&ctype=text/javascript"></script>


    <!-- favicons
	================================================== -->
    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
    <link rel="icon" href="favicon.ico" type="image/x-icon">

</head>

<body id="top">

    <header id="header" class="row">   

		<div class="header-logo">
		<a href="https://2021.igem.org/Team:Phystech_Moscow">Team</a>
		</div>

		<nav id="header-nav-wrap">
			<ul class="header-main-nav">
				<li class="current"><a class="smoothscroll"  href="#home" title="home">Home</a></li>
				<li><a class=""  href="#" title="team">Team</a></li>
				<ul>
					<li><a href="https://2021.igem.org/Team:Phystech_Moscow/Team">About team</a></li>
					<li><a class="imp1" href="https://2021.igem.org/Team:Phystech_Moscow/Attributions">Attributions</a></li>
					<li><a class="imp2" href="https://2021.igem.org/Team:Phystech_Moscow/Collaborations">Collaborations</a></li>
				</ul>
				<li><a class=""  href="#" title="project">Project</a></li>
				<ul>
					<li><a class="imp3" href="https://2021.igem.org/Team:Phystech_Moscow/Communication">Communication</a></li>
					<li><a class="imp1" href="https://2021.igem.org/Team:Phystech_Moscow/Contribution">Contribution</a></li>
					<li><a class="imp1" href="https://2021.igem.org/Team:Phystech_Moscow/Description">Description</a></li>
					<li><a class="imp2" href="https://2021.igem.org/Team:Phystech_Moscow/Engineering">Engineering</a></li>
					<li><a class="imp3" href="https://2021.igem.org/Team:Phystech_Moscow/Model">Model</a></li>                   
					<li><a href="https://2021.igem.org/Team:Phystech_Moscow/Experiments">Experiments</a></li>
					<li><a class="imp2" href="https://2021.igem.org/Team:Phystech_Moscow/Implementation">Implementation</a></li>
					<li><a class="imp3" href="https://2021.igem.org/Team:Phystech_Moscow/Proof_Of_Concept">Proof of сoncept</a></li>
					<li><a href="https://2021.igem.org/Team:Phystech_Moscow/Results">Results</a></li>
					<li><a class="imp3" href="https://2021.igem.org/Team:Phystech_Moscow/Partnership">Partnership</a></li>
				</ul>
				<li><a class=""  href="https://2021.igem.org/Team:Phystech_Moscow/Safety" title="safety">Safety</a></li>
				<li><a class="imp23"  href="https://2021.igem.org/Team:Phystech_Moscow/Human_Practices" title="human_practices">Human practices</a></li>	
			</ul>
			<a href="https://2021.igem.org/Team:Phystech_Moscow/Results" class="button button-primary cta">
				Try it!
			</a>
		</nav>

 		<a class="header-menu-toggle" href="#"><span>Menu</span></a>    	

	</header> <!-- /header -->


   <!-- home
   ================================================== -->
   <section id="home" data-parallax="scroll" data-image-src="https://2021.igem.org/wiki/images/5/5b/T--Phystech_Moscow--Moscow.jpg" data-natural-width=3000 data-natural-height=2000>

        <div class="overlay"></div>
        <div class="home-content">        

            <div class="row contents">                     
                <div class="home-content-left">

                    <h3 data-aos="fade-up">Phystech_Moscow</h3>

                    <h1 data-aos="fade-up">
                        Proof of concept
                    </h1>
                </div>
            </div>
            <ul class="home-social-list">
				<li>
					<a href="https://www.instagram.com/igem_phystech/" target="_blank"><i class="fa fa-instagram"></i></a>
				</li>
				<li>
					<a href="https://web.facebook.com/igemphystech" target="_blank"><i class="fa fa-facebook-square"></i></a>
				</li>
			</ul>

            <div class="home-scrolldown">
				<a href="#styles" class="scroll-icon smoothscroll">
					<span>Down</span>
					<i class="icon-arrow-right" aria-hidden="true"></i>
				</a>
			</div>

        </div> <!-- end home-content -->
    </section>

    <div>
        <p id="img_license" style="padding-left:5% ;font-size: 0.9rem;font-style: italic;"><a href="https://www.flickr.com/photos/95572727@N00/48475349166">"Downtown Moscow"</a><span> by <a href="https://www.flickr.com/photos/95572727@N00">Trey Ratcliff</a></span> is licensed under <a href="https://creativecommons.org/licenses/by-nc-sa/2.0/?ref=ccsearch&atype=html" style="margin-right: 5px;">CC BY-NC-SA 2.0</a></p>
    </div>

    
   <!-- styles
   ================================================== -->
    <section id="styles">
		
        <div class="row narrow add-bottom text-center">

            <div class="col-twelve tab-full">

                <!-- <h1 data-aos="fade-up">This is the style page.</h1> -->

                <p data-aos="fade-up" class="lead">
                    Any computational predictions made in biology need to be validated experimentally to make them trustworthy. It’s great we have developed our prediction model that can help to find miRNA targets of lncRNAs - now its predictions must be checked in the wet lab. Once we make sure that it (hopefully) defines proper miRNA targets, we can say it works well with high probability and produces reliable results. It then can be used by scientists all over the world in their routine lab work, being valuable cost- and time-saving tool in lncRNA studies. 
                    Identification of lncRNA-miRNA interactions, and thus enhanced understanding of lncRNA biology, may shed light on mechanisms of different cellular processes and development of various diseases (diabetes, cardiovascular and neurodegenerative diseases, multiple cancers and etc.). The task is even more challenging and significant due to lncRNAs attractiveness as biomarkers and potential therapeutic targets. 
                    

                </p>   			

            </div>

        </div>

        <div class="row" data-aos="fade-up">

            <div class="col-six tab-full">

                <h3 style="color:white">LncRNAs of interest</h3>
                <p>
                    LncRNAs have been important research objects at Skoltech for a long period of time. In particular, one of lncRNAs studied at prof. Zatsepin’s lab is LINC00152. This lncRNA generally has oncogenic functions and contributes to development of multiple cancers, such as gastric cancer, lung cancer, hepatocellular carcinoma (HCC) and others (1). It was reported to deploy a variety of mechanisms of action in different cancer cells (2). However, exact biological functions of LINC00152 in HCC remain unclear, and one of the deployed mechanisms might be miRNA sponging. 
The researchers are eager to find possible miRNA targets of LINC00152 and then show that lncRNA-miRNA interaction plays role in carcinogenesis of HCC.  Thus, LINC00152 was chosen as lncRNA for experimental validation of the model – in case of successful prediction, the researchers will be able to study importance of LINC00152-miRNA interactions in more detail. 
Another lncRNA we would choose to check for sponging predicted miRNAs is well-characterized MALAT1 (metastasis associated lung adenocarcinoma transcript 1). This lncRNA is involved in a range of physiological processes including alternative splicing and gene expression regulation and was also shown to play significant role in tumorigenesis (particularly metastasis and cancer cell migration), having strong prognostic and therapeutic potential (3). MALAT1 acts as oncogene in many cancers including prostate cancer. Several miRNAs that are sequestered by MALAT1 are already described in the scientific literature – we can check basic performance of our model by comparing its output with previously reported miRNAs, and also perform experiments on newly predicted miRNAs. 
The question remaining is: how can we check if our lncRNAs of interest actually sponge predicted miRNAs? 


                </p>


                <h3 style="color:white">Measure concentration of miRNAs upon knockdown and overexpression of lncRNAs                </h3>
                <p>
                    If lncRNA sponges miRNA, the concentration of free miRNA molecules obviously decreases and is maintained at some relatively low level.
                    <br/>* Thus, if we perform depletion of particular lncRNA to inhibit its activity, we would expect the number of free miRNAs to increase. 
                    <br/>* Reversely, if lncRNA is overexpressed in the cells, more molecules of lncRNAs are produced and more of them bind targeted miRNAs, leading to greater decrease in miRNA concentration. 
                    <br/>Then, we can measure the change in concentration of miRNAs upon depletion and overexpression of lncRNAs with the help of RT-qPCR analysis. 
                    <br/>If we do not see such picture as a result of our experiment, most likely the assumption was wrong and there is no biological interaction between lncRNA of interest and predicted miRNA.
                    <br/>If we do see such change in miRNA concentration upon altering expression of lncRNA, we can consider corresponding miRNA as possible partner of lncRNA.
                    <br/>Such experimental scheme would allow to select best candidates for further studies of lncRNA-miRNA interactions and elucidating their role in development of human diseases. 
                    

                </p>


            </div>

            <div class="col-six tab-full">

                <h3 style="color:white">Protocols</h3>

                <p>
                    For the functional studies of lncRNAs of interest, it is convenient to compare two phenotypes: a cell line with knockdown (KD) of particular lncRNA (its activity is inhibited) and a cell line with overexpression (OE) of the lncRNA (an increased number of lncRNA molecules is produced in cells). A wild-type (WT) cell line is used as a control.
                    <br/>Therefore, for experimental validation of prediction model, cell line with OE and KD of lncRNA of interest must be obtained. Then we can measure miRNA concentration directly with the help of RT-qPCR analysis in three cell lines (WT, KD and OE). 
                    <br/>Cell culture
    <br/>Adherent cell cultures of hepatocellular carcinoma Huh7 (KO/KD, OE and WT) and prostate cancer PC3 (KO/KD, OE and WT) are cultivated in cell culture flasks T-25 in 5 ml of Gibco Dulbecco's Modified Eagle Medium (DMEM) supplemented with 10% fetal bovine serum and 1% penicillin-streptomycin. Cell lines are preserved at 37°C and 5% CO2. Cells are passaged at 85-100% confluency and detached using TrypLE Express Enzyme. In experiments, typically cells of 3-4 passages are used. 
<br/>lncRNA overexpression
<br/>PCR of total cell cDNA (cDNAs produced from all RNA molecules in the sample) is performed with primers specific to lncRNA transcript cDNA and containing restriction sites chosen for cloning. Amplified sequence is purified from the mixture using with Evrogen Cleanup Mini kit. Then cDNA of lncRNA transcript is cloned into mammalian expression vector, e.g. pcDNA3.1. For that, both plasmid and obtained sequence of cDNA undergo restriction digestion with corresponding restriction enzymes at the appropriate conditions. Digested vector and transcript are ligated using Rapid DNA Ligation Kit (Thermo Fisher Scientific) according to manufacturer’s protocol.  
<br/>Afterwards ligation mixture is used for transformation of competent E.coli Mix&Go cells. Bacterial cells are seed on Petry Dish with LB Agar with 100μg/mL of Ampicillin and maintained at 37°C approximately for 16h. 	After PCR-screening of individual clones, clones containing target plasmid are transferred in the 2 ml of LB media and grown overnight. Plasmids are finally extracted with Evrogen Plasmid Miniprep and then verified by sequencing. 
<br/>Obtained plasmids are used for transfection of human cell line with Lipofectamine 2000 Reagent (ThermoFisher Scientific) according to manufacturer’s protocol; cells are grown in media containing antibiotic geneticin (resistance to which is encoded in pcDNA3.1) for selection of successfully transfected cells. Effectiveness of OE can be verified by qPCR method. 
<br/>KD of lncRNA
<br/>A set of antisense oligonucleotides (ASOs) targeting lncRNA transcript should be synthesized and used for transfection of cells with Lipofectamine 2000 Reagent according to manufacturer’s protocol. In that case, lncRNA activity is inhibited due to complementary binding of ASOs, which promotes RNase H activity and degradation of transcript. Success of KD can be verified by qPCR method. 
<br/>miRNA extraction
<br/>Each cell line is grown in three flasks (so that we have three replicates per each). On the fourth passage, cells are centrifuged for 5 minutes at 500 rcf. The growth medium is then removed, cells are washed with PBS buffer and centrifuged at the same conditions; afterwards, PBS buffer is removed from sedimented cells. The subsequent miRNA extraction from each sample is performed in accordance with manufacturer’s guidance of miRNeasy Mini Kit (Qiagen) for miRNA isolation, which allows to extract free miRNAs from cells without extracting other types of RNAs as well as lncRNA-miRNA duplexes.
<br/>RT-qPCR analysis of predicted miRNAs 
<br/>The enzyme reverse transcriptase uses microRNA as a template for the synthesis of the first strand of complementary DNA (cDNA). For microRNA, a hairpin RT primer is used, which is annealed to six 3'-terminal nucleotides of microRNA. The short length of microRNAs does not allow RT-qPCR to be performed according to standard protocols. Due to the structure of the RT hairpin primer, an extended (> 60 nt) first strand of cDNA is synthesized, which can be used as a template for primers annealing in PCR.
<br/>Further, during PCR amplification, a specific forward primer complementary to the microRNA sequence and a universal reverse primer complementary to the hairpin primer sequence are used. RT-qPCR is carried out according to the protocol https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3152947/ 

    
                </p>

                <h3 style="color:white">References </h3>
                <p> 1. Yu Y, Yang J, Li Q, Xu B, Lian Y, Miao L. LINC00152: A pivotal oncogenic long non-coding RNA in human cancers. Cell Prolif. 2017;50(4):1–6.
                    <br/> 2. Xu J, Guo J, Jiang Y, Liu Y, Liao K, Fu Z, et al. Improved characterization of the relationship between long intergenic non-coding RNA Linc00152 and the occurrence and development of malignancies. Cancer Med. 2019;8(10):4722–31.
                    <br/> 3. Arun G, Aggarwal D, Spector DL. MALAT1 long non-coding RNA: Functional implications. Non-coding RNA. 2020;6(2):1–17. </p>
            </div>	         

        </div> <!-- end row -->
     

        <div class="row" data-aos="fade-up" style="align-content: right; padding-right: 5%;">
            <a style="margin-left: auto; margin-right: 0;" class="button button-primary" href="#">Next</a>
        </div>
    </section>


    <!-- footer
    ================================================== -->
    <footer>

        <div class="footer-bottom">
            <div class="row">
                <ul class="footer-social-list">
                    <li>
                        <p>Phystech_Moscow</p>
                    </li>
                    <li>
                        <a href="https://www.instagram.com/igem_phystech/" target="_blank"><i class="fa fa-instagram"></i></a>
                    </li>
                    <li>
                        <a href="https://web.facebook.com/igemphystech" target="_blank"><i class="fa fa-facebook-square"></i></a>
                    </li>
                </ul>
            </div>

            <div class="row">

                <div class="col-twelve">
                    <div class="copyright">
                        <span>© Copyright Phystech_Moscow 2021</span> 
                        <span>© Copyright Dazzle 2018</span> 
                        <span>Design by <a href="http://www.styleshout.com/">styleshout</a></span>		         	
                    </div>

                    <div id="go-top">
                        <a class="smoothscroll" title="Back to Top" href="#top"><i class="icon-arrow-up"></i></a>
                    </div>         
                </div>

            </div> <!-- end footer-bottom -->     	

        </div>

    </footer>

    <script type="text/javascript" src="https://2021.igem.org/wiki/index.php?title=Template:Phystech_Moscow/jquery-2.1.3.min.js&action=raw&ctype=text/javascript"></script>
    <script type="text/javascript" src="https://2021.igem.org/wiki/index.php?title=Template:Phystech_Moscow/plugins.js&action=raw&ctype=text/javascript"></script>
    <script type="text/javascript" src="https://2021.igem.org/wiki/index.php?title=Template:Phystech_Moscow/main.js&action=raw&ctype=text/javascript"></script>
    <script type="text/javascript" src="https://2021.igem.org/wiki/index.php?title=Template:Phystech_Moscow/mine.js&action=raw&ctype=text/javascript"></script>
    <script type="text/javascript">
        AOS.init();
    </script>
    <script type="text/javascript">
        $('#top_title').remove();
        $('#HQ_page').removeAttr('id');
        $('#content').removeAttr('id');
        $('#globalWrapper').removeAttr('id');
        $('#bodyContent').removeAttr('id');
        $('#mw-content-text p').css('margin-bottom', '0px');
    </script>
</body>

</html>